docs/eventing/getting-started.md

```diff
@@ -0,0 +1,438 @@
+
+## Verifying events were received 
+
+After sending events, verify that the events were received by the appropriate `Subscribers`.
+
+1. Look at the logs for the `foo-display` event consumer with the following command:
+
+```sh
+kubectl -n default logs -l app=foo-display
+```
+
+
+The command shows the `Attributes` and `Data` of the events you sent to `foo-display`:
+
+```sh
+☁️  cloudevents.Event
+Validation: valid
+Context Attributes,
+  specversion: 0.2
+  type: foo
+  source: anything-but-bar
+  id: should-be-seen-by-foo
+  time: 2019-05-20T17:59:43.81718488Z
+  contenttype: application/json
+Extensions,
+  knativehistory: default-broker-srk54-channel-24gls.default.svc.cluster.local
+Data,
+  {
+    "msg": "Hello World event from Knative - Foo"
+  }
+☁️  cloudevents.Event
+Validation: valid
+Context Attributes,
+  specversion: 0.2
+  type: foo
+  source: bar
+  id: should-be-seen-by-test
+  time: 2019-05-20T17:59:54.211866425Z
+  contenttype: application/json
+Extensions,
+  knativehistory: default-broker-srk54-channel-24gls.default.svc.cluster.local
+Data,
+  {
+    "msg": "Hello World event from Knative - Both"
+  }
+```
+
+2. Look at the logs for the `bar-display` event consumer with the following command:
+
+```sh
+kubectl -n default logs -l app=bar-display
+```
+
+The command shows the `Attributes` and `Data` of the events you sent to `bar-display`:
+
+
+```sh
+☁️  cloudevents.Event
+ Validation: valid
+ Context Attributes,
+   specversion: 0.2
+   type: anything-but-foo
+   source: bar
+   id: should-be-seen-by-bar
+   time: 2019-05-20T17:59:49.044926148Z
+   contenttype: application/json
+ Extensions,
+   knativehistory: default-broker-srk54-channel-24gls.default.svc.cluster.local
+ Data,
+   {
+     "msg": "Hello World event from Knative - Bar"
+   }
+ ☁️  cloudevents.Event
+ Validation: valid
+ Context Attributes,
+   specversion: 0.2
+   type: foo
+   source: bar
+   id: should-be-seen-by-test
+   time: 2019-05-20T17:59:54.211866425Z
+   contenttype: application/json
+ Extensions,
+   knativehistory: default-broker-srk54-channel-24gls.default.svc.cluster.local
+ Data,
+   {
+     "msg": "Hello World event from Knative - Both"
+   } 
+```
+
+These results should match up with the CloudEvents you created in **Sending CloudEvents to the Broker**. If they do not, see the [Debugging Guide](./debugging/README.md) for more information.
+
+## Cleaning up
+
+Delete the namespace to conserve resources:
+
+```sh
+kubectl delete namespace default
+```
+
+## What’s next 
+
+You've learned the basics of the Knative Eventing workflow. Here are some additional resources to help you continue to build with the Knative Eventing component.
+
+- [Using event importer to consume events](TODO)
+- [Configure `Broker` to Use a Different Channel](https://github.com/Harwayne/knative-eventing/blob/sample-3/configure-broker-to-use-a-different-channel.md)
```

Comment:
For the What's Next section, we don't want to leave in any ToDos and we also don't want to link out to Adam's personal GitHub. We can link to existing content in our documentation or point to how-to guides for other components instead.